### DISCUSSION TITLE
Endpoint não acessivel

### DISCUSSION THREAD

**Discussion by ghost** [balta-io/2811#6] [Q&A] [answered]
Consigo acessar a rota inicial sem problemas:
app.MapGet("/", () => "Hello world");

Quando crio uma nova rota (GET), recebo 404 Not Found:
app.MapGet("/nome", () => "Rota nome");

**Comment by andrebaltieri** [comment, marked as answer]
Boa noite, @vitormancini, como vai?

Preciso de mais detalhes, um trecho de código, URL que o App rodou VS URL que acessou no browser...
Sem mais informações não consigo te dizer o motivo do erro.
